src/sentry/monitors/validators.py

```diff
@@ -230,21 +230,25 @@ def validate(self, attrs):
         return attrs
 
 
-@extend_schema_serializer(exclude_fields=["project", "name", "slug", "config", "alert_rule"])
+@extend_schema_serializer(exclude_fields=["project", "config", "alert_rule"])
 class MonitorValidator(CamelSnakeSerializer, PreventNumericSlugMixin):
     project = ProjectField(scope="project:read")
-    name = serializers.CharField(max_length=128)
+    name = serializers.CharField(
+        max_length=128, help_text="Name of the monitor. Used for notifications."
+    )
     slug = serializers.RegexField(
         DEFAULT_SLUG_PATTERN,
         max_length=MAX_SLUG_LENGTH,
         required=False,
         error_messages={
             "invalid": DEFAULT_SLUG_ERROR_MESSAGE,
         },
+        help_text="Uniquely identifies your monitor within your organization. Changing this slug will require updates to any instrumented check-in calls.",
     )
     status = serializers.ChoiceField(
         choices=list(zip(MONITOR_STATUSES.keys(), MONITOR_STATUSES.keys())),
         default="active",
+        help_text="One of [active, disabled]. Disabled monitors do not generate events or notifications.",
```

Comment:
I think the API spec already shows the options, let's leave it off of the help text since we would have to keep it in sync otherwise.

Reply:
I wonder if it's possible to create documentation for the enums themselves?

Reply:
I don't see it in the docs now https://docs.sentry.io/api/crons/create-a-monitor/

Reply:
isn't that it right here

![image](https://github.com/getsentry/sentry/assets/1421724/8dab2f1d-dd5a-42eb-82dc-51ead49c99a8)

?

Reply:
Ah right was looking in the wrong place